recipe/0002-anaconda-win-library-bin-path.patch

```diff
@@ -0,0 +1,33 @@
+From 6f5dd2e40cc5cf77fddbbf709a266d13f9ba46fd Mon Sep 17 00:00:00 2001
+From: Mike Sarahan <msarahan@gmail.com>
+Date: Mon, 1 Jul 2019 09:35:31 -0500
+Subject: [PATCH] anaconda win library bin path
+
+---
+ libarchive/ffi.py | 3 +++
+ 1 file changed, 3 insertions(+)
+
+diff --git a/libarchive/ffi.py b/libarchive/ffi.py
+index 755eb71..292b88d 100644
+--- a/libarchive/ffi.py
++++ b/libarchive/ffi.py
+@@ -15,6 +15,7 @@ from ctypes.util import find_library
+ import logging
+ import mmap
+ import os
++import sys
+ 
+ from .exception import ArchiveError
+ 
+@@ -24,6 +25,8 @@ logger = logging.getLogger('libarchive')
+ page_size = mmap.PAGESIZE
+ 
+ libarchive_path = os.environ.get('LIBARCHIVE') or find_library('archive')
++if sys.platform == 'win32' and not libarchive_path:
++    libarchive_path = os.path.join(sys.prefix, 'Library', 'bin', 'libarchive.dll')
```

Comment:
Do we also want to provide paths for other platforms?
```
if sys.platform == 'darwin':
    libarchive_path = os.path.join(sys.prefix, 'lib', 'libarchive.dylib')
if sys.platform.startswith('linux'):
    libarchive_path = os.path.join(sys.prefix, 'lib', 'libarchive.so')
```

Reply:
sure, doesn't hurt, I guess.  It's just a fallback, after all.